src/components/SideBar/SideBar.jsx

```diff
@@ -1,101 +1,87 @@
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
+import { useNavigate, useLocation } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import {
-  Wrapper,
-  ProfileBox,
-  LogOutBox,
-  Frame,
-  // ProfileBoxAvatar,
-  // LogOutButton,
-} from './SideBarStyled';
+import { Wrapper, ProfileBox, LogOutBox, Frame } from './SideBarStyled';
 
 import { getUserName } from '../../redux-store/AuthOperations/selectors';
 import { logOut } from '../../redux-store/AuthOperations/AuthOperations';
 // import Icons from '../Icons/Icons';
 import SideBarItem from '../SideBarItem/SideBarItem';
 
-function SideBar({ onClickDms, onClickRooms, isActive }) {
+const SideBar = ({ onClickDms, onClickRooms, isActive }) => {
   const userName = useSelector(getUserName);
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const location = useLocation();
+  const currentPage = location.pathname;
 
   const handleProfileOpen = () => {
-    navigate('/app/account');
-    console.log('clicked profile');
+    navigate('account');
```

Comment:
Use routesPath constant for navigating between routes.

Reply:
Done